module/merintr/mermain.c

```diff
@@ -134,13 +164,9 @@ void InterfaceRedrawModule(HDC hdc)
   StatsDrawBorder();
   StatsMainRedraw();
   StatsDraw();
-  if( StatsGetCurrentGroup() == STATS_INVENTORY )
-  {
-    InvalidateRect( GetHwndInv(), NULL, FALSE );
-    ShowInventory(TRUE);
-    InventoryRedraw();
-  }
+  RestoreActiveStatGroup();
```

Comment:
I'm still confused by the logic here.  This is in the window redraw function.  We potentially need to fetch stats from the server every time the window is redrawn?  Seems like there could be some cases where the client can generate tons of messages if the client is (e.g.) resized while we're waiting to get the stats back from the server.

Is the idea that RestoreActiveStatGroup() needs to be called after a system save?  Every redraw is overkill for this obviously.  I wonder if there's a function that's called less often, or another mechanism for hooking into restores after system saves.